packages/transactions/lib/transactions.js

```diff
@@ -143,6 +143,8 @@ export class Transactions {
    * @param {string} account
    */
   async loadExternalTransactions(account) {
+    assert(typeof account === 'string', 'invalid account address');
```

Comment:
empty, or all whitespace? Is there a particular account string format we could assert? Maybe a helper validation function `checkAccountString(account)` that could be re-used throughout?

Reply:
I have some concerns with performance if we test an account address with much details. But testing empty, or all whitespace would be good